cli/module_loader.rs

```diff
@@ -115,6 +117,56 @@ impl CliModuleLoader {
       )),
     }
   }
+
+  fn load_sync(
+    &self,
+    specifier: &ModuleSpecifier,
+  ) -> Result<ModuleSource, AnyError> {
+    let code_source = if self.ps.npm_resolver.in_npm_package(specifier) {
+      let file_path = specifier.to_file_path().unwrap();
+      let code = std::fs::read_to_string(file_path)?;
+      let is_cjs = self.ps.cjs_resolutions.lock().contains(specifier);
+
+      // translate cjs to esm if it's cjs
+      let code = if is_cjs {
+        node::translate_cjs_to_esm(
+          &self.ps.file_fetcher,
+          specifier,
+          code,
+          MediaType::Cjs,
+          &self.ps.npm_resolver,
+        )?
+      } else {
+        node::esm_code_with_node_globals(specifier, code)?
+      };
```

Comment:
At the moment, no source mapping modifications are done based on the modifications that happen here. That will be done in the future.